admin/widgets.ts

```diff
@@ -24,6 +24,11 @@ export type SiteRoute = string;
  */
 export type MapWidget = string;
 
+/**
+ * @format date-time
+ */
+export type DateTime = string;
+
```

Comment:
Add date as another possible type. Also, add 'widget' to avoid confusion with base types (`Date`).

```suggestion
/**
 * @format date
 */
export type DateWidget = string;

/**
 * @format date-time
 */
export type DateTimeWidget = string;

```